### DISCUSSION TITLE
Documentation struggles

### DISCUSSION THREAD

**Discussion by mmontag** [discordjs/discord.js#9839] [General]
### Which application is this bug report for?

Documentation

### Issue description

Hi all, new to discord.js and really like it so far. However, I'm struggling with the discord.js documentation. 
Two things in particular:

- The documentation for the current discord.js release version 14 loads up in old.discordjs.dev. The domain name is confusing. (Hopefully obsoleted soon.)
- Search is...yikes, it's bad. For example, one might wonder how to format messages sent by a bot. Let's search for "formatting": https://old.discordjs.dev/#/docs/discord.js/14.11.0/search?query=formatting 
  - the results are a mix of 100+ methods, classes, types with no context
  - sometimes the search doesn't do anything, not sure if it's broken
  - can't tell what part of the results matched my query
  - whole site appears uncrawled by Google, so e.g. this doesn't work: https://www.google.com/search?q=site%3Aold.discordjs.dev+formatting

### Steps to Reproduce

none

### Versions

none

### Issue priority

Low (slightly annoying)

**Comment by iCrawl** [comment]
Noted.

1. This is intended. It is the "old" or legacy version of discord.js. It doesn't mean that its outdated or anything, it's still the latest version as of right now.
2. The documentation you are on is an API reference, as others call it. It is automatically generated. It is not meant to work as a guide or have searchable articles that show you how to "use" stuff. That is what the guide is for: https://discordjs.guide/

**Comment by danielgwilson** [comment]
+1, the documentation is really confusing—I'm still unclear; is it recommended we use `@discordjs/core` or `discordjs`? 🤔 🤯

**Reply by tipakA** [reply]
`/core` is a subpackage that doesn't have all the "hand holding" you get with `discord.js`. If you know what you're doing, or know why would you want to, you're probably good to go with `/core`, otherwise `discord.js` is most likely the one you should use.

That said, the `old.` subdomain is overall confusing. The reason that's done this way is because the new docs only work with TypeScript code, and discord.js itself is the only thing that is still in JavaScript. The reasoning shouldn't really be "old version of d.js" but "old version of docs", because there *isn't* a newer version of discord.js (guide has a `v13` subdomain for "legacy" content if you need it, but the main address displays current content).

When The Eventual TypeScript Rewrite™️ of discord.js will happen, it will be on the new docs as discord.js just fine, alongside all the other subpackages.

**Comment by alaanvv** [comment]
The documentation is really worse than ever, but there's this version who at least tries to be as good as it used to be

Well there was I link here that I just searched fast and wrong pasted, I can't find the good docs anymore ToT

**Reply by Qjuh** [reply]
We‘re always happy about feedback. What parts of the new docs do you deem *worse than ever* and how are the old docs better in that aspect?

**Reply by alaanvv** [reply]
Well, I'll ignore the fact I loved that old style.

Maybe for you the points I'm going to say are all refutable, but know that I'm just an average user with 2IQ

Besides the style, the search system doesn't seem to be straight, it feels like sometimes I'm typing an exact thing and that isn't the first item of the results
2. The navbar also seems to be with too much (I know it's supposed to have everything, but maybe there's a solution for this)

That's what I can notice, but it clearly feels slower than the old one
